submitter/toil_submitter/toil_jobsubmitter.py

```diff
@@ -227,6 +227,8 @@ def _command_line(self):
                 "--disable-host-provenance",
                 "--stats",
                 "--debug",
+                "--disableProgress",
+                "--doubleMem",
```

Comment:
We might also consider making use of the `--statePollingWait` parameter here

Reply:
Good idea!